import-export-cli/cmd/mg/login.go

```diff
@@ -0,0 +1,63 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"github.com/spf13/cobra"
+	impl "github.com/wso2/product-apim-tooling/import-export-cli/impl/mg"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var loginUsername string
+var loginPassword string
+var loginPasswordStdin bool
+
+const loginCmdLiteral = "login [environment]"
+const loginCmdShortDesc = "Login to an Microgateway Adapter environment"
```

Comment:
```suggestion
const loginCmdShortDesc = "Login to a Microgateway Adapter environment"
```
Shall we fix this typo?

Reply:
Thanks, updating this.